wazimap_np/templates/homepage.html

```diff
@@ -24,14 +24,14 @@ <h3>
                     <i class="fa fa-book"></i>
                     Find facts
                 </h3>
-                <p>On the beta version of NepalMap, population data is available. Soon more data will be added. </p>
+                <p>On the beta version of NepalMap, demographics, agriculture, education and households data are available.</p>
```

Comment:
Consider using adjectives for all of the four categories:
"demographic, agricultural, educational and household data"
To me, "demographic, agricultural, education and household data" also sounds okay, but I think "educational" is probably better

Reply:
ok will change to educational